openslides_backend/action/actions/chat_message/create.py

```diff
@@ -0,0 +1,49 @@
+from time import time
+from typing import Any, Dict
+
+from ....models.models import ChatMessage
+from ....shared.exceptions import PermissionDenied
+from ....shared.patterns import Collection, FullQualifiedId
+from ...generics.create import CreateAction
+from ...util.default_schema import DefaultSchema
+from ...util.register import register_action
+
+
+@register_action("chat_message.create")
+class ChatMessageCreate(CreateAction):
+    """
+    Action to create a chat message.
+    """
+
+    model = ChatMessage()
+    schema = DefaultSchema(ChatMessage()).get_create_schema(
+        required_properties=["chat_group_id", "content"],
+    )
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        instance["user_id"] = self.user_id
+        instance["meeting_id"] = self.get_meeting_id(instance)
```

Comment:
You can use the `CreateActionWithInferredMeeting` here

Reply:
I use the meeting inferring action now. It simplifies the code.